pyflakes/checker.py

```diff
@@ -1307,12 +1307,19 @@ def _enter_annotation(self, ann_type=AnnotationState.BARE):
 
     @contextlib.contextmanager
     def _exit_annotation(self):
-        orig, self._in_annotation = self._in_annotation, False
+        orig, self._in_annotation = self._in_annotation, AnnotationState.NONE
```

Comment:
this can reuse `_enter_annotation(ann_type=AnnotationState.NONE)`

Reply:
Done.